src/desktop/DesktopDownloadManager.js

```diff
@@ -93,10 +94,10 @@ export class DesktopDownloadManager {
 			return Promise.reject('canceled')
 		}
 
-		const downloadItem = new DownloadItem()
+		const downloadItem: DownloadItem = new EventEmitter()
 		downloadItem.getFilename = () => filename
 
-		this._handleDownloadItem(new Event("saveBlob"), downloadItem)
+		this._handleDownloadItem(downcast("saveBlob"), downloadItem)
```

Comment:
I checked and this event is not even needed. Should we get rid of it? @ganthern

Reply:
seems like it has outlived its usefulness